Your pull request does not meet our code style rules. Pylint summary:
```
************* Module lb
testsuite/python/lb.py:575:12: W0612: Unused variable 'i' (unused-variable)
```

You can generate these warnings with `maintainer/CI/fix_style.sh`. This is the same command that I have executed to generate the log above.

--- a/src/core/lb/particle_coupling.cpp
+++ b/src/core/lb/particle_coupling.cpp
@@ -251,7 +251,6 @@ void ParticleCoupling::kernel(Particle &p) {
   Utils::Vector3d force_on_particle = {};
   auto folded_pos = box_geo.folded_position(p.pos());
 
-
 #ifdef ENGINE
   if (not p.swimming().is_engine_force_on_fluid)
 #endif

--- a/testsuite/python/lb.py
+++ b/testsuite/python/lb.py
@@ -565,17 +565,16 @@ class LBTest:
                 [n.last_applied_force for n in all_coupling_nodes])
             np.testing.assert_allclose(
                 np.sum(applied_forces, axis=0), [0, 0, 0])
-    
+
     def test_viscous_coupling_rounding(self):
         lbf = self.lb_class(**self.params, **self.lb_params)
         self.system.lb = lbf
         self.system.thermostat.set_lb(LB_fluid=lbf, seed=3, gamma=self.gamma)
-        p=self.system.part.add(pos=[-1E-30]*3,v=[-1,0,0])
+        p = self.system.part.add(pos=[-1E-30] * 3, v=[-1, 0, 0])
         self.system.integrator.run(1)
-        for i in range(20):
+        for _ in range(20):
             self.system.integrator.run(1)
             self.assertTrue(np.all(p.f != 0.0))
-
 
     def test_thermalization_force_balance(self):
         system = self.system

